src/openforms/authentication/contrib/digid_oidc/plugin.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any, Dict, Optional
+from urllib.parse import urlencode
+
+from django.http import HttpRequest, HttpResponseBadRequest, HttpResponseRedirect
+from django.templatetags.static import static
+from django.utils.translation import gettext_lazy as _
+
+import requests
+from furl import furl
+from rest_framework.reverse import reverse
+
+from openforms.authentication.base import BasePlugin, LoginLogo
+from openforms.authentication.registry import register
+from openforms.forms.models import Form
+
+from ...constants import CO_SIGN_PARAMETER, FORM_AUTH_SESSION_KEY, AuthAttribute
+from ...exceptions import InvalidCoSignData
+from .constants import DIGID_OIDC_AUTH_SESSION_KEY
+from .models import OpenIDConnectPublicConfig
+
+
+@register("digid_oidc")
+class DigiDOIDCAuthentication(BasePlugin):
+    verbose_name = _("DigiD via OpenID Connect")
+    provides_auth = AuthAttribute.bsn
+
+    def start_login(self, request: HttpRequest, form: Form, form_url: str):
+        login_url = reverse("digid_oidc:oidc_authentication_init", request=request)
+
+        auth_return_url = reverse(
+            "authentication:return",
+            kwargs={"slug": form.slug, "plugin_id": "digid_oidc"},
+        )
+        # The return_url becomes the DigiD relay state - this is why we add the co-sign
+        # param to that URL and not the `form_url`.
```

Comment:
Comment is not updated to the specific plugin - we don't have a relay state concept with oidc